config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'tasks/index'
```

Comment:
@kaitq-afterfit 
chỗ này không có to controller à e?

Reply:
Controller và routing hiện tại em mới đang code nên lúc trước design static layout thì em chỉ dùng lệnh generate của rails (rails tự thêm dòng get 'tasks/index') ý anh. Lúc đó em chưa để ý file route ạ.